pcb/rules/F6_2.py

```diff
@@ -49,9 +41,13 @@ def check(self):
         THRESHOLD = 0.001
         if abs(x) > THRESHOLD or abs(y) > THRESHOLD:
             self.error("Footprint anchor is not located at center of footprint")
-            self.errorExtra("Footprint center calculated as ({x},{y})mm".format(
-                x = round(x, 5),
-                y = round(y, 5)))
+            self.errorExtra("Footprint center calculated as Pad: ({xp},{yp})mm F.Fab: ({xf}, {yf})mm".format(
+                xp = round(center_pads['x'], 5),
+                yp = round(center_pads['y'], 5),
+                xf = round(center_fab['x'], 5),
+                yf = round(center_fab['y'], 5)))
+            if center_pads['x'] != center_fab['x'] or center_pads['y'] != center_fab['y']:
+                self.errorExtra("Footprint centers of pads and F.Fab do not match")
```

Comment:
This message is still here. It still suggests that the user needs for some reason ensure pads and fab need to be on the same center.